### DISCUSSION TITLE
How to get completion code in Phoenix code for Python?

### DISCUSSION THREAD

**Discussion by Cezanna** [phcode-dev/phoenix#1607] [Q&A]
Hi, 
Everything is in the title .....

**Comment by charlypa** [comment]
we have code completions available for html and javascript. You have to press `ctrl + space` to trigger completion.

**Comment by Cezanna** [comment]
May I hope completion for python in the future, and if yes, when ?
Thank you

**Reply by abose** [reply]
Python code completion should be available once we have language servers integrated some time between August-October.
